packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -254,6 +257,28 @@ class ExpansionTile extends StatefulWidget {
   ///   [ExpansionTileThemeData].
   final Color? collapsedTextColor;
 
+  /// Tile's border shape when the sublist is expanded.
+  ///
+  /// If this property is null then [ExpansionTileThemeData.shape] is used. If that
+  /// is also null then a [Border] with vertical direction sides default to [ThemeData.dividerColor] is used
```

Comment:
I am not sure I follow this, does this mean there is only a border on the sides?

Reply:
I've used this default 'cause it was the design of the expansion tile before this change and I don't want to break the previous changes.

i.e, the expansion tile in the flutter has its vertical border set to ThemeData.dividerColor